kafka-connect-mysql/variant.mod

```diff
@@ -0,0 +1,23 @@
+provisioners:
+  files:
+    Dockerfile:
+      source: Dockerfile.tpl
+      arguments:
+        kafka_connect_version: "{{ .kackaConnect.version }}"
+        mysql_connector_version: "{{ .mysqlConnector.version }}"
+
+dependencies:
+  kackaConnect:
+    releasesFrom:
+      dockerImageTags:
+        source: confluentinc/cp-kafka-connect-base
+    version: "< 6.0.0"
```

Comment:
Since only `ubi8` images are provided with kafka connect series 6, we restrict ourselves to using only series 5.
6系のバージョンを利用するさいに`Dockerfile`を更新する予定です。